api/src/main/java/org/openmrs/api/ProgramWorkflowService.java

```diff
@@ -43,22 +43,22 @@
  * </pre>
  */
 public interface ProgramWorkflowService extends OpenmrsService {
-	
+
```

Comment:
I wouldnt add in these changes. I would only expect to see the changes you made putting in spaces reverted by changing them to tabs.

Reply:
@teleivo I converted all the spaces I had added to tabs, hence the above changes